.github/workflows/ethereum-tests.yml

```diff
@@ -29,11 +29,20 @@ jobs:
         with:
           cache-on-failure: true
 
-      - name: Run Ethereum tests
+      - name: Run Ethereum tests (Debug mode)
         run: |
           cargo run --profile ethtests -p revme -- \
           statetest \
           ethtests/GeneralStateTests/ \
           ethtests/LegacyTests/Constantinople/GeneralStateTests/ \
           tests/EIPTests/StateTests/stEIP5656-MCOPY/ \
           tests/EIPTests/StateTests/stEIP1153-transientStorage/
+
+      - name: Run Ethereum tests (Release mode)
+        run: |
+          cargo run --release -p revme -- \
+          statetest \
+          ethtests/GeneralStateTests/ \
+          ethtests/LegacyTests/Constantinople/GeneralStateTests/ \
+          tests/EIPTests/StateTests/stEIP5656-MCOPY/ \
+          tests/EIPTests/StateTests/stEIP1153-transientStorage/
```

Comment:
These tests already take a while to compile and run, can you make it a matrix so the two profiles run in parallel?

Reply:
Great suggestion! Now it's 2x faster 💪